index.html

```diff
@@ -23,10 +25,11 @@
         </header>
         <main> 
             <section class="banner-container">
-                <!-- I'm struggling to get CSS to modify the img -->
+               
                     <h2>Friday, plan your next week</h2>
                     <img src="images/image_banner.png" alt="Happy lady smiling at shopfront">
```

Comment:
According to the initial wireframe we created, in the banner, the idea was to have a heading and a "create" button on top of a background image. It could be much harder to include an image in the the HTML comparing to setting a background image via CSS. So, could you please work it out via the CSS?

Reply:
I just noticed that in the css rule set, .banner-container img {, the same image is set as background-image, which is not neccessary. So, when you revise your code, please also take it out.